channels/jo.m3u

```diff
@@ -15,8 +15,28 @@ https://jrtv-live.ercdn.net/jordanhd/jordanhd.m3u8
 https://cdn3.wowza.com/1/ZFBldUlPNjRBRDZM/ZW90V2ZW/hls/live/playlist.m3u8
 #EXTINF:-1 tvg-id="NojoumTV.jo" tvg-country="JO" tvg-language="Arabic" tvg-logo="https://i.imgur.com/6ttWNu0.jpg" group-title="Music",Nojoum TV (720p) [Offline]
 http://51.210.199.41/hls/stream.m3u8
-#EXTINF:-1 tvg-id="RotanaRadio.jo" tvg-country="JO" tvg-language="Arabic" tvg-logo="http://service.mobile.rotanaradio.jo/sites/default/files/studio.png" group-title="Music",Rotana Radio (220p) [Not 24/7]
+#EXTINF:-1 tvg-id="" tvg-country="JO" tvg-language="Arabic" tvg-logo="" group-title="Religious",Radio Yaqeen Jordan: Control Studio
+http://188.247.86.66/MixFM/ControlStudio1/playlist.m3u8
```

Comment:
Not 24/7 I shall assume. Got nothing right now.

Reply:
Fixed